### PR TITLE
Footer and Github Link

### PR DESCRIPTION
**Pull request by cste303**
### Changes

Added a footer that includes a message about font storage and a link to the GitHub.

- <bullet point changes>

### Tests applied

What testing has been applied to validate changes are accurate and correct

- Link to GitHub works
- Visual testing

### Issue ticket number(s)

Enter the issue numbers resolved by this pull request

- #51 

### Checklist

- [ ] Documented
- [ ] Linting tests successful
- [ ] merge updated prod branch into development branch

### CODE REVIEW

File: src/styles/App.css [hunk 1]
```diff
@@ -31,3 +31,17 @@ main {
...
+.github-icon {
+    font-size: 1.6rem;
+    color: black;
+}
+
+footer {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    background-color: #f0f0f0;
```
Comment (tpo920) [inline src/styles/App.css]: I think we can add this colour to the `index.css` section and use it from there

**Review by tpo920 — COMMENTED** [review]
Looks good to me Connor, just a minor detail about the `background-color` in `App.css`, instead of using the hexadecimal value, maybe create and reference the colour from `index.css`

File: src/styles/App.css (hunk 1 above)
Comment (cste303) [reply src/styles/App.css]: Good idea I have moved it now.

**Review by tpo920 — APPROVED** [review]
LGTM!

File: src/index.css [hunk 2]
```diff
@@ -9,6 +9,7 @@
     --color-fg-variant: rgb(60% 60% 60%);
     --color-border: rgb(86% 86% 86%);
     --color-accent: rgb(39.22% 42.35% 100%);
+    --color-footer-bg: #f0f0f0;
```
Comment (jaskfla) [inline src/index.css]: We may want to use this elsewhere. Perhaps we call it `—color-bg-variant` so the variable name isn’t coupled to a particular element?

And I’ll leave you to decide if you want to change it to `rgb(94.12% 94.12% 94.12%)` for consistency.

**Review by jaskfla — APPROVED** [review]
Looking good! Just a small nitpick about variable naming, but I don’t actually mind if it makes it into the production branch so will approve.

Fix as you see fit and feel free to merge.

File: src/index.css (hunk 2 above)
Comment (cste303) [reply src/index.css]: I can make those changes probably not a bad idea.

**Review by jaskfla — APPROVED** [review]
Looking great—I’m happy to merge. Thanks!

**Pull request merged**